.orchestra/support/revng-distributable/install.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "Copying root"
+rm -rf "${DESTDIR}${ORCHESTRA_ROOT}/revng/root"
+mkdir -p "${DESTDIR}${ORCHESTRA_ROOT}/revng/root"
+
+cd "$ORCHESTRA_ROOT"
+
+orc inspect component dependencies --runtime revng-distributable | \
```

Comment:
Didn't we say to just have `--build-only` to show only build deps? Or is specifying either `--runtime` or `--build` mandatory?

Reply:
* No switch => both build and runtime
* `--runtime` => only runtime

Reply:
Ah right!